packages/backend/src/config/config.testing.ts

```diff
@@ -12,5 +13,6 @@ export function getTestConfig(): Config {
       format: 'json',
     },
     port: 1337,
+    databaseUrl: getEnv('TEST_DB_URL', 'xXTestDatabaseUrlXx'),
```

Comment:
Improve the name, as the other comment suggests

Reply:
What do you feel about this import I just introduced?

Reply:
I like it, makes it clear where the value is coming from